bindata/network/ovn-kubernetes/common/004-rbac-control-plane.yaml

```diff
@@ -156,6 +156,24 @@ rules:
     - ipamclaims/status
   verbs: [ "patch", "update" ]
 {{- end}}
+{{- if .OVN_NETWORK_SEGMENTATION_ENABLE }}
+- apiGroups:
+    - discovery.k8s.io
+  resources:
+    - endpointslices
```

Comment:
this would also give us power to change eps created by primary k8s controller a well, we need to be careful but we are consciously doing this